### DISCUSSION TITLE
cn/2024/12/18/videos/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#198] [Announcements]
# cn/2024/12/18/videos/

https://hongtaoh.com/cn/2024/12/18/videos/

**Comment by earfanfan** [comment]
那小孩子们咋办呢？很多小孩会说会听的时候，其实还不认字的说。

**Reply by hongtaoh** [reply]
那么小，就好好享受童年吧。不要卷了。
